webdriver-spec.html

```diff
@@ -1231,6 +1237,15 @@
  </tr>
 
  <tr>
+  <td><dfn>no such cookie</dfn>
```

Comment:
Unfortunately the cookie stuff never errors, unless you are doing cross domain, why are we introducing this when it not how the platform works and breaking change?